### ISSUE TITLE
Something is odd about `runScriptWithArgs` signature

### ISSUE THREAD

**Issue by noomorph** [wix/Detox#4127]
### What happened?

@elgrogz reports:

> We're trying to use the runScriptWithArgs webview function to access a webview's window object but we're having a bit of difficulty. Ideally we'd just run the script against the entire webview but the runScriptWithArgs seems to be run against a particular element in the webview. Is that correct? Maybe then this isn't a valid use case for the function. If anyone knows how we can get the window object then please let me know 🙂 This is our current attempt:

```
send(params: Record<string, unknown>) {
    const webview = web.element(by.web.id("root"));
    const id = randomUUID();
    const json = JSON.stringify({
      id,
      jsonrpc: "2.0",
      ...params,
    });

    webview.runScriptWithArgs(
      `function(json, element) {
        alert("yo");
        window.ledger.e2e.walletApi.send('${json}');
        alert("yoo");
      }`,
      [json],
    );
  }
```

I also noticed that the expected arguments don't match up with what the docs say: <https://wix.github.io/Detox/docs/api/webviews/#runscriptwithargsscript-args>

It says it expects ...args which implies any amount of args, but it actually expects an array of any[]. When I pass an array I get an Unhandled arg typeArrayList from Java. And if I use the exact function in the docs it says there are too many arguments. Is this a bug, and if not how should I be handling it?

![image](https://github.com/wix/Detox/assets/1962469/7cdbe452-1845-4f8d-af18-dd97c3f7e5af)
![image](https://github.com/wix/Detox/assets/1962469/cf1807a4-258b-4225-b074-2b7f8b5930f0)

### Was it tested on latest Detox?

- [ ] I have tested this issue on the latest Detox release and it still reproduces.

### In what environment did this happen?

Detox version: 
React Native version: 
Has Fabric (React Native's new rendering system) enabled: (yes/no) 
Node version: 
Test-runner (select one): jest / other

### Detox logs

<details><summary>Detox logs</summary>

```
paste logs here!
```

</details>

### Device logs

<details><summary>Device logs</summary>

```
paste logs here!
```

</details>

**Comment by 17Amir17** [comment]
Seeing this too

**Comment by pagman77** [comment]
Also running into the same issue here.

**Comment by noomorph** [comment]
The fix is on its way. I left you a question for you here regarding the updated signature: https://github.com/wix/Detox/pull/4221

**Comment by noomorph** [comment]
✅  **The fix is available in [`detox@20.13.1`](https://github.com/wix/Detox/releases/tag/20.13.1).**

**Comment by noomorph** [comment]
@elGrogz, @17Amir17, @pagman77, @ggilchrist-ledger, please check out the new version.
